Editor/Animation/AnimatorCombiner.cs

```diff
@@ -277,13 +281,15 @@ private void AdjustBehavior(StateMachineBehaviour behavior)
         {
             switch (behavior)
             {
+#if MA_VRCSDK3_AVATARS
```

Comment:
Move this outside the switch for now

Reply:
Should I rather wrap entire `AdjustBehavior()` method and calls to it, or keep the `AdjustBehavior()` method empty?

Reply:
Leave the method empty for now, I think it's reasonable to keep the method call itself to avoid spreading `#if`s all over the codebase